Added Jamie Tanna's go tag feed

dotemacs opened the pull request:
On his blog he has various topics, but the Golang specific ones are tagged with `go` tag, which is nice of him.

That way only Golang relevant posts will be featured on the planet.

Here is his `go` feed, for a quick browse: https://www.jvt.me/tags/go/

/cc @jamietanna

dotemacs committed:
Added Jamie Tanna's go tag feed

On his blog he has various topics, but the Golang specific ones are
tagged with `go` tag, which is nice of him.

That way only Golang relevant posts will be featured on the planet.

Here is his `go` feed, for a quick browse: https://www.jvt.me/tags/go/

jamietanna commented:
Thanks very much, and thanks for sharing another great resource for me to read :grin:

jamietanna commented:
Hopefully it'll be OK with the mix of articles and other things in there :crossed_fingers:

dotemacs commented:
> Hopefully it'll be OK with the mix of articles and other things in there 🤞

It should be, since I've only added the feed that pulls your posts tagged with **go** only.

That way the planet will only source the relevant posts from your blog.

dotemacs commented:
That is a good question, since not many people/blogs post/share other people's posts that they like...

I'm not sure what would be a good solution?

Maybe you can have something on your end that is "only go blog posts of yours" feed?
Not sure if such a thing is possible with your blog setup...

jamietanna commented on feeds.yml:
```diff
@@ -157,4 +157,9 @@ rednafi:
+  scraper: FeedScraper
+jvt:
+  title: Jamie Tanna | Software Engineer
+  scrape_url: https://www.jvt.me/tags/go/feed.xml
```
I've been meaning to create an articles-only feed page for tags, so this has been a good nudge to get into doing it!

I've just pushed the changes, which should mean you can now pull:

```suggestion
  scrape_url: https://www.jvt.me/tags/go/feed.articles.xml
```

And this will only list blog posts.

This will be live in ~10-20 minutes ([build+deploy job](https://gitlab.com/tanna.dev/jvt.me/-/pipelines/1251096602)) at which point we should be good to merge this!

dotemacs replied:
That's super cool of you @jamietanna  😄  thanks

@tomasurbanec Let me know what you think, thanks

tomasurbanec replied:
Hi,
sorry for long time without response.
I will happily merge it. Thanks, both of you, for thinking it through so well. :+1:

dotemacs committed:
Updated Jamie Tanna's go tag feed

Jamie was kind enough to create a new RSS feed, prompted by this PR,
that only lists his blog posts.

Because on this blog, he reviews other people's blog posts, which also
get a `go` tag. This way, with this new feed, only his blog posts will
be provided.

Thanks Jamie :)
